src/scripts/controllers/container.controller.js

```diff
@@ -56,6 +56,9 @@
             this.ngModel = {};
           } else if (this.ngModel instanceof Date) {
             this.ngModel.setHours(0, 0, 0, 0);
+          } else if (typeof(this.ngModel) === 'string' || this.ngModel instanceof String) {
```

Comment:
I think that this two conditions test the same situation. A simple `else if (typeof this.ngModel === 'string')` would suffice.